src/mats_l1_processing/read_parquet_functions.py

```diff
@@ -227,10 +270,23 @@ def read_ccd_data_in_interval(
                 & (ds.field(variable) <= filter[variable][1])
             )
 
+    st = time.time()
+    table = dataset.to_table(
+        filter=partition_filter & filterlist
+    )
```

Comment:
```suggestion
    table = dataset.to_table(filter=partition_filter & filterlist)
```
looks neater, I think.

Reply:
ok fixed